requirements.txt

```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
+openwisp-utils[rest]
 influxdb>=5.2,<5.3
-django-notifications-hq>=1.6,<1.7
-django-celery-email>=3.0.0,<3.1
-djangorestframework>=3.11,<3.12
+django-notifications-hq~=1.6
```

Comment:
let's replace this with `openwisp-notifications~=0.1`, which has been released, related to https://github.com/openwisp/openwisp-monitoring/issues/231. 

I would not remove it from install-dev yet.

Reply:
I haven't tagged the issue (as to be closed) as we are not yet removing the openwisp-notifications `dev` (which makes sense given the addition of many new changes).